website/src/client/configs/sdk.tsx

```diff
@@ -12,5 +12,5 @@ export const versions: Record<SDKVersion, boolean> = {
   '42.0.0': true,
 };
 
-export const DEFAULT_SDK_VERSION = '41.0.0';
-export const TEST_SDK_VERSION = '39.0.0';
+export const DEFAULT_SDK_VERSION: SDKVersion = '42.0.0';
```

Comment:
snack-sdk also defines a "default" SDK-version. This const has always confused me a bit and feels duplicate. In my opinion we can just remove this and use the default SDK version from `snack-sdk`. @byCedric what do you think?

Reply:
The only scenario where this could go wrong is when we release an `x.x.x-rc.0` version. That shouldn't be set on the website instantly right? When we do that, we have to be able to use a different default version on the website

Reply:
Yeah I think you're right. Lets keep it as is 👍